### DISCUSSION TITLE
Merchant details are there when Searching under Product Set and adding manually in Product set.

### DISCUSSION THREAD

**Discussion by pirajki** [datafeedr/wordpress-plugins#24] [Help]
Merchant details are there when Searching under Product Set and adding manually in Product set. But it is not showing in Bulk editor.

I am using these codes 

function mycode_normalize_merchant_name( $value, $attribute, $post, $product, $set, $action ) {

	$ai = new Dfrpswc_Attribute_Importer( 'store', $value, $attribute, $product );

	$ai->add_field( [ "merchant" ], "store", "other" );

	return $ai->result();
}

add_filter( 'dfrpswc_filter_attribute_value', 'mycode_normalize_merchant_name', 20, 6 );

**Comment by EricBusch** [comment]
Hi

I see you have the attribute set to `store`. Do you also have an product attribute named "Store" here WordPress Admin Area > Products > Attributes?

Eric

**Comment by pirajki** [comment]
Hi

Yes we have

**Comment by EricBusch** [comment]
After you added that code did you also [Bump](https://datafeedrapi.helpscoutdocs.com/article/46-update-a-product-set) your Product Sets?

**Comment by pirajki** [comment]
Yes

**Comment by EricBusch** [comment]
Is your Custom Code plugin activated?

**Comment by pirajki** [comment]
Yes

**Comment by EricBusch** [comment]
Can you send a screenshot of your attributes page here WordPress Admin Area > Products > Attributes?

**Comment by pirajki** [comment]
It is exporting merchant for some products and blank in others, how ever
the products are having merchant attribute and value in them

**Comment by EricBusch** [comment]
> But it is not showing in Bulk editor.

What bulk editor are you referring to? Can you send a screenshot?

> It is exporting merchant for some products and blank in others

Our plugins do not export data, they only import data. We do not have an export feature.

> the products are having merchant attribute and value in them

Do you mean the data from Datafeedr? If yes, that is correct. Every product has a `merchant` field.

**Comment by pirajki** [comment]
in WooCommerce Advanced Bulk Edit(v5.0.1)
we are checking list of products, and there we can see these missing
values, which we can also check in individual, product details/. this store
attribute is not importing properly

**Comment by EricBusch** [comment]
Hi

When you view the Product Edit page, are you seeing the Merchant's name listed in the "Attributes" section of the "edit" page?

Thanks
Eric
